jetson/percep/main.cpp

```diff
@@ -256,18 +252,19 @@ int main() {
     lcm_.publish("/target_list", &arTagsMessage);
     lcm_.publish("/obstacle", &obstacleMessage);
 
-    #if PERCEPTION_DEBUG
-      imshow("depth", depth_img);
-      waitKey(FRAME_WAITKEY);
+    #if PERCEPTION_DEBUG && AR_DETECTION
+      imshow("depth", src);
+      waitKey(1);
+      for(int i = 0; i < 10000; ++++i){
```

Comment:
It's a temporary fix that slows down the image read, that way people actually have time to see what the image is. If there was no loop the program would appear to fail since it would just flash a screen and then end. It's a temporary fix. I have people working on developing a better solution

Reply:
https://en.cppreference.com/w/cpp/thread/sleep_for

Fix your temporary solution before merging. We do not use for loops to re-implement sleeping